manifests/config.pp

```diff
@@ -63,15 +63,16 @@
   contain foreman_proxy::module::httpboot
 
   contain foreman_proxy::module::puppet
-  foreman_proxy::provider { [
-      'puppet_proxy_customrun',
-      'puppet_proxy_mcollective',
-      'puppet_proxy_puppet_api',
-      'puppet_proxy_salt',
-      'puppet_proxy_ssh',
-    ]:
+  foreman_proxy::provider { [ 'puppet_proxy_puppet_api', ]:
   }
-  foreman_proxy::provider { ['puppet_proxy_legacy', 'puppet_proxy_puppetrun']:
+  foreman_proxy::provider { ['puppet_proxy_legacy',
```

Comment:
I think that having every element of the array on a separate line is cleaner.

Reply:
Just to be clear, you're proposing making this array similar to the one that used to be above?  i.e.:
```suggestion
  foreman_proxy::provider { [
    'puppet_proxy_legacy',
```

Reply:
Yes